doc/shared_content/how_release.rst

```diff
@@ -0,0 +1,52 @@
+How to Release?
+===============
+
+Creating a Release
+++++++++++++++++++
+
+#. Prepare the project for a new release:
+
+    .. code-block:: shell
+
+        nox -s release:prepare -- [-h] [-v | --version VERSION] [-t | --type {major,minor,patch}]
+
+#. Merge your **Pull Request** to the **default branch**
+
+#. Trigger the release:
+
+    .. code-block:: shell
+
+        nox -s release:trigger
+
+What to do if the release failed?
++++++++++++++++++++++++++++++++++
+
+The release failed during pre-release checks
+--------------------------------------------
+
+#. Delete the local tag
+
+    .. code-block:: shell
+
+        git tag -d "${TAG}"
```

Comment:
maybe change as tag not set anymore as bash variable in top instructions
cp. `TAG="<major>.<minor>.<patch>"`